Training25/Program.cs

```diff
@@ -3,11 +3,44 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T11 - Armstrong number.
 // ------------------------------------------------------------------------------------------------
 namespace Training25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      string input = GetInput ();
+      ArmStrong (input);
    }
-}
+   /// <summary> Gets only postive integer from the user </summary>
+   static string GetInput () {
+      int n = 0;
+      while (true) {
+         Console.Write ("Enter a positive integer: ");
+         string? input = Console.ReadLine (); //Can accept null value
+         if (string.IsNullOrWhiteSpace (input)) {
+            Console.WriteLine ("Input cannot be empty");
+            continue; //Asks user again for input
+         }
+         bool isValid = int.TryParse (input, out n); //Checks whether the input is int
+         if (!isValid || n < 0) {

```

Comment:
`isValid` used once so it can be written as :
```suggestion
         if (!int.TryParse (input, out n) || n < 0) {
```